_posts/en/newsletters/2021-04-21-newsletter.md

```diff
@@ -0,0 +1,221 @@
+    the receiver are fully secure and the only consequence is that the
+    spender will need to wait a while before they can reuse that
+    particular one of their payment slots ([HTLC][topic htlc] slots).
+    This ability to communicate interactively is a clear advantage of
+    offers over plain invoices.
+
+- **Using anchor outputs by default in LND:** Olaoluwa Osuntokun
+  [posted][osuntokun anchor] to the LND engineering mailing list about
+  his desire for the next major version of LND to use [anchor
+  outputs][topic anchor outputs] by default.  Anchor outputs will allow
+  unconfirmed LN commitment transactions that close a channel to be
+  [CPFP][topic cpfp] fee bumped.  Unfortunately, there are some
+  challenges with CPFP fee bumping in the LN model:
+
+    - *Not always optional:* for regular onchain transactions, many
+      users can just wait longer for their transaction to confirm as an
+      alternative to fee bumping.  For LN, sometimes waiting isn't an
+      option---a fee bump will need to be submitted within a matter of
+      hours or funds could be lost.
+
+    - *Timelocked outputs:* for most regular onchain payments, a user
+      who wants to CPFP bump can pay for a fee bump using the funds
+      stored in their output from the transaction they want to bump.  In
+      the case of LN, those funds aren't available until the channel
+      close has been fully settled onchain.  That means the user needs
+      to use a separate UTXO to pay the fees.
+
+    To address tho above two concerns, LND is requiring users of anchor
+    outputs to retain at least one confirmed UTXO of reasonable value in
+    their wallet any time a channel is open.  That ensures they can CPFP
+    fee bump when necessary, but it has certain consequences, such as
+    preventing spending the last of your onchain funds (even to open a
+    new channel) while you still have at least one channel open.
+
+    Osuntokun's request is for wallets or services built on LND to let
+    the development team know if any of the above concerns, or any other
+    concerns related to anchor outputs, will cause serious problems.
+    Although the question is specific to LND, the answers may have
+    implications for all LN nodes.
+
+- **Sapio public launch:** Jeremy Rubin [posted][rubin sapio] to the
+  Bitcoin-Dev mailing list an announcement that he has made available
+  the Sapio smart contract development toolkit, a Rust-based library
+  and associated tooling that can be used to create smart contracts
+  expressible using Bitcoin Script.  The language was originally
+  designed to make use of Rubin's proposed
+  [OP_CHECKTEMPLATEVERIFY][topic op_checktemplateverify] opcode
+  (`OP_CTV`) but it can simulate the existence of that opcode, and of
+  other potential features for Bitcoin such as taproot, using a trusted
+  signing oracle.  In addition to the Sapio library, the release also
```

Comment:
```suggestion
  (`OP_CTV`), but it can simulate the existence of that opcode, and offers
  other potential features for Bitcoin such as taproot using a trusted
  signing oracle.  In addition to the Sapio library, the release also
```